prepare-vms/README.md

```diff
@@ -14,7 +14,7 @@ And if you want to generate printable cards:
 ## General Workflow
 
 - fork/clone repo
-- set required environment variables for AWS
+- set required environment variables (for AWS; placeholders otherwise)
```

Comment:
What about "set a few required environment variables"?

Since this is the overview, we could keep it simple, especially since things are detailed a bit lower. WDYT?